framework/CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+Enh #18821: Allow `yii\db\ExpressionInterface` as column in `yii\db\conditions\InBuilder`
```

Comment:
I have once fixed the changelog you made previously like that - please put it in the right place.